.circleci/config.yml

```diff
@@ -107,26 +120,60 @@ jobs:
       - run: sudo make -C build install
       - run: make -C build package
 
+  build-arm-release:
+    docker:
+      - image: arm64v8/ubuntu:23.04
+    resource_class: arm.xlarge
+    steps:
+      - checkout
+      - run: apt update && apt install sudo
+      - linux_preflight
+      - check_ci_lint
+      - linux_deps
+      - run: git submodule sync && git submodule update --init
+      - restore_cache:
+          keys:
+            - ccache-release-linux-arm_64-v3-{{ .Branch }}-{{ checksum "conanfile.txt" }}
+            - ccache-release-linux-arm_64-v3-{{ checksum "conanfile.txt" }}
+      - run: |
+          mkdir build
+          source venv/bin/activate
+          cd build
+          cmake .. -DENABLE_SINGLE_FILES_WERROR=Off -DBENCHMARK_ENABLE_WERROR=Off -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=On -DENABLE_PYTHON_BINDINGS=On \
```

Comment:
two main differences for arm here: `-DENABLE_SINGLE_FILES_WERROR=Off -DBENCHMARK_ENABLE_WERROR=Off`

we cannot enable warnings as errors. for whatever reason the compiler is extra pedantic on arm and so we cant get through a build with werror turned on for pretty much any parts of the project